client/onboarding/style.scss

```diff
@@ -27,10 +30,12 @@ body.wcpay-onboarding__body {
 					display: flex;
 					align-items: center;
 					padding: $gap-large;
+					color: #0073aaff;
```

Comment:
That's not a valid HEX color :)

Reply:
Thanks, I'm not sure where that color came from. I'm now using the color defined in the Figma.